test/06_locales.js

```diff
@@ -28,11 +28,53 @@ var moment = require('moment');
 var range = require('moment-range');
 var should = require('should');
 
-var Locales = require('../index');
+var Romcal = require('../index');
+var Utils = Romcal.Utils;
 
 describe('Testing localization functionality', function() {
 
   this.timeout(0);
 
+  // Locales extended with sample data
+  Romcal.Locales.frCA = Romcal.Locales.frCA || {};
+  Romcal.Locales.frCA.test = { hello: 'Allo', foo: 'foo' };
+  Romcal.Locales.en.test = { hello: 'Hello', foo: 'bar' };
+  Romcal.Locales.fr.test = { hello: 'Salut', lorem: 'ipsum' };
+
+  it('If locale is set to "fr", romcal should output text in French', function() {
+    Utils.setLocale('fr');
+    Utils.localize({key: 'test.hello'}).should.equal('Salut');
+  });
+
+  it('If locale is set to "fr-CA", romcal should output text in French-Canadian', function() {
+    Utils.setLocale('fr-CA');
+    Utils.localize({key: 'test.hello'}).should.equal('Allo');
+  });
+
+  it('If locale is set with an unknown region, romcal should fallback to its base language when exists in src/locales', function() {
+    Utils.setLocale('fr-XX');
+    Utils.localize({key: 'test.hello'}).should.equal('Salut');
+  });
+
+  it('If a string is missing in the "fr-CA" locale, romcal should fallback to base French', function() {
```

Comment:
`fallback` vs `fall back`
`checkout` vs `check out`
`rollback` vs `roll back`

What’s the difference? The former is a noun, the latter is a verb. :smiley:

Note that when you want to use past tense, the verb is not _rollbacked_, but _rolled back_.

Same change should be made in strings bellow, too.

>   it('If a string is missing in the "fr-CA" locale, romcal should fall back to base French', function() {

Reply:
I believe you’ve missed this one. :smiley:

Reply:
I've fixed that on, check the last commit. This one is outdated.

Reply:
Sorry, somehow I missed that. :)